Makefile

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
weak vote against having a Makefile (either the Makefile is super simple and thus not necessary or it is complex and a total monstrosity because, well, it's a Makefile :).